scripts/deploy.sh

```diff
@@ -32,7 +32,11 @@ if [ "$TRAVIS_TAG" == "" ]; then
   echo "Skipping release deployment: not a tag"
 else
   echo "Deploy to bintray..."
-  ./gradlew bintrayUpload -Pbintray.user="${BINTRAY_USER}" -Pbintray.apikey="${BINTRAY_API_KEY}"
+  ./gradlew publishDefaultPublicationToBintrayRepository -Pbintray.user="${BINTRAY_USER}" -Pbintray.apikey="${BINTRAY_API_KEY}"
   echo "Deployed to bintray!"
+  echo "Deploy gradle plugin marker to bintray..."
+  # We override the artifact_id for the marker else it is uploaded at the same coordinates as apollo-gradle-plugin
+  ./gradlew publishApolloGradlePluginPluginMarkerMavenPublicationToBintrayRepository -Pbintray.user="${BINTRAY_USER}" -Pbintray.apikey="${BINTRAY_API_KEY}" -PPOM_ARTIFACT_ID=com.apollographql.apollo.gradle.plugin
```

Comment:
Wouldn't it be possible to keep the artifact id in gradle files?

Reply:
Not really, the marker is a special case since it belongs to the `apollo-gradle-plugin` module.
The problem is that the repository depends on the publication and the `maven-publish` plugin is designed for these things to be independent. We could also duplicate the bintray repository, that would be an option but it'll require slightly more changes